Assertion error on teleport

I was killing Hellhounds in Witchhaven. I downloaded and activated the plugin while doing so. I teleported to Varrock using the normal spellbook. I got an error:

```
[Client] ERROR net.runelite.client.RuneLite - Uncaught exception:
java.lang.AssertionError: null
	at ejedev.chompyhunter.ChompyHunterPlugin.onNpcSpawned(ChompyHunterPlugin.java:94)
	at net.runelite.client.eventbus.EventBus$Subscriber.invoke(EventBus.java:73)
	at net.runelite.client.eventbus.EventBus.post(EventBus.java:222)
	at net.runelite.client.util.DeferredEventBus.replay(DeferredEventBus.java:69)
	at net.runelite.client.callback.Hooks.clientMainLoop(Hooks.java:183)
	at client.my(client.java:21613)
	at client.ao(client.java)
	at br.ab(br.java:362)
	at br.run(br.java:316)
	at java.lang.Thread.run(Thread.java:748)
```

--- a/src/main/java/ejedev/chompyhunter/ChompyHunterPlugin.java
+++ b/src/main/java/ejedev/chompyhunter/ChompyHunterPlugin.java
@@ -80,7 +80,7 @@ public class ChompyHunterPlugin extends Plugin{
     }
 
     @Subscribe
-    private void onNpcSpawned(NpcSpawned event)
+   private void onNpcSpawned(NpcSpawned event)
     {
         NPC npc = event.getNpc();
 
@@ -91,21 +91,22 @@ public class ChompyHunterPlugin extends Plugin{
 
         String name = event.getNpc().getName();
 
-        assert name != null;
-        if (name.equals("Chompy bird") && !chompies.containsKey(npc.getIndex()))
-        {
-            chompies.put(npc.getIndex(), new Chompy(npc));
+        if (name != null) {
+            if (name.equals("Chompy bird") && !chompies.containsKey(npc.getIndex())) {
+                chompies.put(npc.getIndex(), new Chompy(npc));
+            }
         }
     }
 
-    @Subscribe
+   @Subscribe
     private void onNpcDespawned(NpcDespawned event)
     {
         NPC npc = event.getNpc();
         String name = event.getNpc().getName();
-        assert name != null;
-        if (name.equals("Chompy bird") && chompies.containsKey(npc.getIndex())) {
-            chompies.remove(event.getNpc().getIndex());
+        if (name != null) {
+            if (name.equals("Chompy bird") && chompies.containsKey(npc.getIndex())) {
+                chompies.remove(event.getNpc().getIndex());
+            }
         }
     }
 }

